pkgs/by-name/ry/rygel/package.nix

```
@@ -67,13 +69,14 @@ stdenv.mkDerivation (finalAttrs: {
     gupnp-dlna
     libgee
     libsoup_3
-    gtk3
     libmediaart
     sqlite
     systemd
     tinysparql
     shared-mime-info
-  ] ++ (with gst_all_1; [
+  ]
+  ++ lib.optional withGtk gtk3
```

Please use the following optionals is confusing to people and not extensible:

```suggestion
  ++ lib.optionals withGtk [ gtk3 ]
```

---

Thanks, that really helps to know. Done!